lib/middleware/routes-metadata/routes-metadata.js

```diff
@@ -292,6 +292,11 @@ async function pageHandler (req, res) {
 
     pageInstance.sessionDuration = getInstanceProperty('service', 'sessionDuration')
 
+    pageInstance.showTimeoutWarning = true
+    if (pageInstance._type && (pageInstance._type === 'page.start' || pageInstance._type === 'page.confirmation')) {
```

Comment:
we might also want to do `page.content` to cover any static pages like cookies, accessibility etc

Reply:
or any static page that a form owner decides to add

Reply:
Done! ♻️